src/composables/multisigAccountCreate.ts

```diff
@@ -163,17 +182,43 @@ export function useMultisigAccountCreate({ store }: IDefaultComposableOptions) {
       multisigAccountCreationPhase.value = MULTISIG_CREATION_PHASES.deployed;
       const gaContract = await sdk.getAccount(accountId);
       multisigAccountCreationPhase.value = MULTISIG_CREATION_PHASES.created;
+
       multisigAccount.value = {
         contractId: gaContract.contractId,
-        multisigAccountId: accountId,
+        balance: new BigNumber(gaContract.balance),
+        gaAccountId: accountId,
+        signers,
+        confirmationsRequired,
+        consensusLabel: prepareConsensusLabel(0, confirmationsRequired, signers.length),
+        nonce: 1, // Default value for freshly created account
+        createdAt: dayjs().toISOString(),
+        updatedAt: dayjs().toISOString(),
```

Comment:
The `dayjs()` function is fired twice - I suggest to create const for this before the assignment.